elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -481,7 +483,7 @@ See mixin blocks before selectors that contain the selector name and the three l
 // Back to menu icon
 @mixin pfe-navigation__back__before--mobile {
   position: absolute;
-  top: 24px;
+  top: 27px;
```

Comment:
It'd be nice to document this 'magic number' with a comment.

e.g.
```
  top: 27px; // The height of .foo element
```

Reply:
@wesruv made this change